<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TimerCfger</class>
 <widget class="QFrame" name="TimerCfger">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>381</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Frame</string>
  </property>
  <property name="frameShape">
   <enum>QFrame::StyledPanel</enum>
  </property>
  <property name="frameShadow">
   <enum>QFrame::Raised</enum>
  </property>
  <widget class="QRadioButton" name="upTimer">
   <property name="geometry">
    <rect>
     <x>152</x>
     <y>0</y>
     <width>97</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Up Timer</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="downTimer">
   <property name="geometry">
    <rect>
     <x>272</x>
     <y>0</y>
     <width>105</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Down Timer</string>
   </property>
  </widget>
  <widget class="QLabel" name="targetTimeTag">
   <property name="geometry">
    <rect>
     <x>16</x>
     <y>40</y>
     <width>129</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Target Time:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QTimeEdit" name="targetTime">
   <property name="geometry">
    <rect>
     <x>288</x>
     <y>40</y>
     <width>111</width>
     <height>32</height>
    </rect>
   </property>
   <property name="displayFormat">
    <string>HH:mm:ss</string>
   </property>
  </widget>
  <widget class="QDateEdit" name="targetDate">
   <property name="geometry">
    <rect>
     <x>152</x>
     <y>40</y>
     <width>129</width>
     <height>32</height>
    </rect>
   </property>
   <property name="displayFormat">
    <string>yyyy/MM/dd</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="day">
   <property name="geometry">
    <rect>
     <x>152</x>
     <y>80</y>
     <width>65</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Day</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="hour">
   <property name="geometry">
    <rect>
     <x>224</x>
     <y>80</y>
     <width>65</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Hour</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="minute">
   <property name="geometry">
    <rect>
     <x>304</x>
     <y>80</y>
     <width>73</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Minute</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="second">
   <property name="geometry">
    <rect>
     <x>392</x>
     <y>80</y>
     <width>73</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Second</string>
   </property>
  </widget>
  <widget class="QLabel" name="timerFontAttr">
   <property name="geometry">
    <rect>
     <x>15</x>
     <y>120</y>
     <width>129</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Font:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="timenNumColorTag">
   <property name="geometry">
    <rect>
     <x>16</x>
     <y>160</y>
     <width>129</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Number Color:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="unitColorTag">
   <property name="geometry">
    <rect>
     <x>16</x>
     <y>200</y>
     <width>129</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Unit Color:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QCheckBox" name="enabledText">
   <property name="geometry">
    <rect>
     <x>153</x>
     <y>240</y>
     <width>89</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>EnabledText</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineText">
   <property name="geometry">
    <rect>
     <x>152</x>
     <y>272</y>
     <width>297</width>
     <height>32</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="textAttrTag">
   <property name="geometry">
    <rect>
     <x>15</x>
     <y>309</y>
     <width>129</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>TextAttr:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="italic">
   <property name="geometry">
    <rect>
     <x>185</x>
     <y>341</y>
     <width>33</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="../res/vshow.qrc">
     <normaloff>:/image/text_italic_64.png</normaloff>:/image/text_italic_64.png</iconset>
   </property>
  </widget>
  <widget class="QPushButton" name="bold">
   <property name="geometry">
    <rect>
     <x>152</x>
     <y>341</y>
     <width>33</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="../res/vshow.qrc">
     <normaloff>:/image/text_bold_64.png</normaloff>:/image/text_bold_64.png</iconset>
   </property>
  </widget>
  <widget class="QLabel" name="textTag">
   <property name="geometry">
    <rect>
     <x>15</x>
     <y>272</y>
     <width>129</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Text:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QCheckBox" name="newline">
   <property name="geometry">
    <rect>
     <x>272</x>
     <y>240</y>
     <width>65</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Newline</string>
   </property>
  </widget>
  <widget class="QFontComboBox" name="timeFont">
   <property name="geometry">
    <rect>
     <x>152</x>
     <y>120</y>
     <width>177</width>
     <height>32</height>
    </rect>
   </property>
  </widget>
  <widget class="QFontComboBox" name="textFont">
   <property name="geometry">
    <rect>
     <x>152</x>
     <y>308</y>
     <width>177</width>
     <height>32</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>280</number>
   </property>
  </widget>
  <widget class="QPushButton" name="textColor">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>308</y>
     <width>33</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="../res/vshow.qrc">
     <normaloff>:/image/color_text_64.png</normaloff>:/image/color_text_64.png</iconset>
   </property>
  </widget>
  <widget class="QPushButton" name="timeNumColor">
   <property name="geometry">
    <rect>
     <x>152</x>
     <y>160</y>
     <width>33</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="../res/vshow.qrc">
     <normaloff>:/image/color_text_64.png</normaloff>:/image/color_text_64.png</iconset>
   </property>
  </widget>
  <widget class="QPushButton" name="timeUnitColor">
   <property name="geometry">
    <rect>
     <x>152</x>
     <y>200</y>
     <width>33</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="../res/vshow.qrc">
     <normaloff>:/image/color_text_64.png</normaloff>:/image/color_text_64.png</iconset>
   </property>
  </widget>
  <widget class="QLabel" name="timerFontAttr_2">
   <property name="geometry">
    <rect>
     <x>328</x>
     <y>120</y>
     <width>48</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Size:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="underLine">
   <property name="geometry">
    <rect>
     <x>218</x>
     <y>341</y>
     <width>33</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="../res/vshow.qrc">
     <normaloff>:/image/text_underline_64.png</normaloff>:/image/text_underline_64.png</iconset>
   </property>
  </widget>
  <widget class="QSpinBox" name="timeFontSize">
   <property name="geometry">
    <rect>
     <x>376</x>
     <y>120</y>
     <width>48</width>
     <height>32</height>
    </rect>
   </property>
  </widget>
  <widget class="QSpinBox" name="textFontSize">
   <property name="geometry">
    <rect>
     <x>376</x>
     <y>308</y>
     <width>48</width>
     <height>32</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="../res/vshow.qrc"/>
 </resources>
 <connections/>
</ui>
